lib/generators/shopify_app/shop_model/templates/db/migrate/create_shops.rb

```diff
@@ -1,12 +1,14 @@
 class CreateShops < ActiveRecord::Migration
   def self.up
     create_table :shops  do |t|
-      t.string :shopify_domain, null: false
-      t.string :shopify_token, null: false
+      t.string  :shopify_domain, null: false
+      t.string  :shopify_token, null: false
+      t.integer :associated_user_id
+      t.text    :extra_json
```

Comment:
These 2 fields do not belong to `shop` because many users from the same shop may belong to a single shop. I think this should either be in a separate table, or simply kept in the user's session.

Is there a reason you needed to keep this in the db? If not, I would recommend to simply use the session storage.